provider-directory/src/main/docker/entrypoint.sh

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env bash
+
+ENDPOINT_DOMAIN_NAME="$K8S_LOAD_BALANCER"
+ENVIRONMENT="$K8S_ENVIRONMENT"
+BASE_PATH="$BASE_PATH"
+IDENTIFIER="$IDENTIFIER"
+NAME="$NAME"
+GIVEN="$GIVEN"
+
+#Current Version(s) available
+VERSION="/api
```

Comment:
Since we're not going to have the version in the URL like in CCE, let's remove this and just hardcode `/api` down below where it is used